src/Microsoft.Health.Fhir.Api/Features/Routing/KnownRoutes.cs

```diff
@@ -58,6 +58,9 @@ internal class KnownRoutes
         public const string Everything = "$everything";
         public const string PatientEverythingById = "Patient/" + IdRouteSegment + "/" + Everything;
 
+        public const string PurgeHistory = "$purge-history";
+        public const string PurgeHistoryResourceTypeById = ResourceTypeById + "/" + PurgeHistory;
```

Comment:
>PurgeHistoryResourceTypeById [](http://example.com/codeflow?start=28&length=28)

To make it proper operations we need:
-Capability statement
-json file describing operation
-route which returns that json

Reply:
I think I mentioned that we should look at making operations a little simpler to add :)

Reply:
Were they made simpler to add? What is the minimum required to add an operation, if all of the FHIR resources are available?

When asking the question I was pointed here. But this seems to have additional changes that may not be required.